tests/test_ote_api.py

```diff
@@ -167,28 +180,168 @@ def test_sample_on_gpu(self):
         assert output.returncode == 0
 
 
-class TestOTEAPI(unittest.TestCase):
+class APITestCase(unittest.TestCase):
     """
     Collection of tests for OTE API and OTE Model Templates
     """
 
+    @staticmethod
+    def generate_random_annotated_image(
```

Comment:
I would propose to move this function from old SDK test helpers to new SDK test helpers

Reply:
You can even do that, I guess 😉